assets/css/draw.css

```diff
@@ -0,0 +1,133 @@
+html {
+    margin: 0;
+    padding: 0;
+  }
+  
+  body {
+    height: 100vh;
+    width: 100vw;
+    overflow-y: hidden;
```

Comment:
You can simply write it `overflow: hidden` .

Reply:
Yeah, that can also be done. Should I change that right away?

Reply:
Yes !!